File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/Content.java
package com.porkandlager.travelonbudget.wires.models.beans;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tista on 11/12/16.
 */

public class Content {

    @SerializedName("wikipedia_url")
    private String wikipediaUrl;

    @SerializedName("location")
    private Location location;

    @SerializedName("picture")
    private String pictureUrl;

    @SerializedName("description")
    private String description;

    public String getWikipediaUrl() {
        return wikipediaUrl;
    }

    public Location getLocation() {
        return location;
    }

    public String getPictureUrl() {
        return pictureUrl;
    }

    public String getDescription() {
        return description;
    }

    public void setWikipediaUrl(String wikipediaUrl) {
        this.wikipediaUrl = wikipediaUrl;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public void setPictureUrl(String pictureUrl) {
        this.pictureUrl = pictureUrl;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/Content.kt
package com.porkandlager.travelonbudget.wires.models.beans

import com.google.gson.annotations.SerializedName

/**
 * Created by tista on 11/12/16.
 */

class Content {

    @SerializedName("wikipedia_url")
    var wikipediaUrl: String? = null

    @SerializedName("location")
    var location: Location? = null

    @SerializedName("picture")
    var pictureUrl: String? = null

    @SerializedName("description")
    var description: String? = null
}
